oclint-reporters/reporters/HTMLReporter.cpp

```diff
@@ -123,44 +127,53 @@ class HTMLReporter : public Reporter
         out << "<head>";
         out << "<title>OCLint Report</title>";
         out << "<style type='text/css'>"
-            << "                             \
-.priority1, .priority2, .priority3,          \
-.cmplr-error, .cmplr-warning, .checker-bug { \
-    font-weight: bold;                       \
-    text-align: center;                      \
-}                                            \
-.priority1, .priority2, .priority3 {         \
-    color: #BF0A30;                          \
-}                                            \
-.priority1 { background-color: #FFC200; }    \
-.priority2 { background-color: #FFD3A6; }    \
-.priority3 { background-color: #FFEEB5; }    \
-.cmplr-error, .cmplr-warning {               \
-    background-color: #BF0A30;               \
-}                                            \
-.cmplr-error { color: #FFC200; }             \
-.cmplr-warning { color: #FFD3A6; }           \
-.checker-bug {                               \
-    background-color: #002868;               \
-    color: white;                            \
-}                                            \
-table {                                      \
-    border: 2px solid gray;                  \
-    border-collapse: collapse;               \
-    -moz-box-shadow: 3px 3px 4px #AAA;       \
-    -webkit-box-shadow: 3px 3px 4px #AAA;    \
-    box-shadow: 3px 3px 4px #AAA;            \
-}                                            \
-td, th {                                     \
-    border: 1px solid #D3D3D3;               \
-    padding: 4px 20px 4px 20px;              \
-}                                            \
-th {                                         \
-    text-shadow: 2px 2px 2px white;          \
-    border-bottom: 1px solid gray;           \
-    background-color: #E9F4FF;               \
-}"
+            << "* { color: #4A4A4A ; } \
+.sortable { cursor: pointer; } \
```

Comment:
It might be tedious, but could you align the backslash, please?

Reply:
will do